apps/explorer/lib/explorer/exchange_rates/source.ex

```diff
@@ -4,9 +4,56 @@ defmodule Explorer.ExchangeRates.Source do
   """
 
   alias Explorer.ExchangeRates.Token
+  alias HTTPoison.{Error, Response}
 
   @doc """
-  Callback for fetching an exchange rates for currencies/tokens.
+  Method fetching exchange rates for currencies/tokens.
```

Comment:
1. It's not acallback (as it's not part of a `@behaviour` `@impl`
2. It's also not a method because Elixir is a functional language, there are only functions, not methods.
3. In general, starting docs with "Callback..." or "Method..." where the type is repeated in the docs should be discouraged.

Instead, describe what it does only.

```suggestion
  Fetches exchange rates for currencies/tokens.
```